Automattic/jetpack
# Boost Speed Score: Fix PHP 8.2 deprecation notice

## Proposed changes:

* Fixes dynamic property deprecation notice, due to undeclared property.

I fixed the notice, but after trying to tail where the property is used, I kind of lost track and couldn't pinpoint if it should be there at all. So I just left the property in.

```
[14-Sep-2023 15:01:47 UTC] PHP Deprecated:  Creation of dynamic property Automattic\Jetpack\Boost_Speed_Score\Speed_Score_Graph_History_Request::$status is deprecated in /usr/local/src/jetpack-monorepo/projects/packages/boost-speed-score/src/class-speed-score-graph-history-request.php on line 129
```

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Use PHP 8.2+ when testing;
* Enable WP debug;

### Test that it breaks
* Setup latest boost or trunk;
* Setup Boost and access the dashboard (free or premium, doesn't matter);
* Check `debug.log` and there should be a PHP deprecation notice about creating a dynamic property (status).

### Test that it works
* Setup this branch;
* Setup Boost and access the dashboard (free or premium, doesn't matter);
* The notice shouldn't show up.
